java/src/main/java/com/powsybl/dataframe/BaseDataframeMapperBuilder.java

```diff
@@ -22,6 +25,7 @@
 
     protected Function<T, List<U>> itemsProvider;
     protected BiFunction<T, String, U> itemGetter;
```

Comment:
We should remove that one, since it is replaced by the new more generic getter.

In order to keep the existing `itemGetter` method, you need to change its implementation so that it creates the new kind of getter fro the older simpler one.